homeassistant/components/xs1.py

```diff
@@ -0,0 +1,117 @@
+"""
+Support for the EZcontrol XS1 gateway.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/xs1/
+"""
+
+import asyncio
+import logging
+from functools import partial
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_HOST, \
+    CONF_PORT, CONF_USERNAME, CONF_PASSWORD
+from homeassistant.helpers import discovery
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['xs1-api-client==2.3.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'xs1'
+ACTUATORS = 'actuators'
+SENSORS = 'sensors'
+
+# configuration keys
+CONF_SSL = 'ssl'
+
+# define configuration parameters
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_HOST): cv.string,
+        vol.Optional(CONF_PORT): cv.string,
```

Comment:
If there is a standard port then that port should be the default.

Reply:
ok